.github/workflows/go.yml

```diff
@@ -22,9 +22,9 @@ jobs:
       uses: golangci/golangci-lint-action@9d1e0624a798bb64f6c3cea93db47765312263dc # v5.1.0
       with:
         # Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
-        version: v1.55.2
+        version: v1.58.1
         # only-new-issues is only applicable to PRs, otherwise it is always set to false
-        only-new-issues: false # disabled for PRs due to unreliability
+        only-new-issues: true
```

Comment:
There is too much to fix, so I think we need to flip this back and hope for the best :crossed_fingers: